sources/Licenses/README.md

```diff
@@ -0,0 +1,31 @@
+## Motivation
+
+Licenses, such as those from [Creative Commons](https://creativecommons.org/licenses/), are often miswritten and mismatches between version statements (2.0 or v2) are not easy to track. The Licenses.yml rule aims to comprehensively solve these problems.
+
+We preferred to adopt the list of Debian-compatible licenses due to the fact that Debian, as a reliable organization, is also relatively selective in the licenses it accepts. Furthermore, the information provided with the [Debian Free Software Guidelines](https://wiki.debian.org/DebianFreeSoftwareGuidelines) (DFSG) is accessible and permissively licensed.
```

Comment:
What do think about referring to this?

https://spdx.org/licenses/

Not saying the Debian one is wrong, but to mention this one also exist.

I'm unsure how they differs

Reply:
I'm unsure as to the license of the SPDX list. Likely, GPLv2, but I haven't been able to confirm.

I, in part, chose the Debian compatible list because it seems to be appropriately selective. I think that should be preferable for us in this case to prevent potential intersections with future tech term or abbreviation trype rules. You can check the table [here](https://en.wikipedia.org/wiki/Comparison_of_free_and_open-source_software_licenses#Approvals).

I preferred Debian's list over Fedora because their list is under CC BY-SA and includes superseded licenses. The GNU list of GPLv3 compatibility on the other hand doesn't include fairly popular licenses such as Eclipse and GPLv2.

Thus, the Debian list was ideal since they were selective and include nearly all popular licenses. Furthermore, they are a reliable organization with a clear process of vetting licenses which offloads that burden from us.

Reply:
I'll add that Wikipedia link to the motivation part.